docs/guides/code/max_tasks_per_minute.py

```diff
@@ -0,0 +1,23 @@
+import asyncio
```

Comment:
Could you please rename the file to `scaling_crawlers_max_tasks_per_minute`?

Reply:
Updated